src/wechaty_puppet_itchat/puppet.py

```diff
@@ -175,9 +172,7 @@ def __init__(self, options: PuppetOptions = None, name: str = 'puppet_itchat'):
         self.login_user_id: Optional[str] = None
         self.puppet_options = None
         self.puppet = self
-        self.itchat = itchat
-        self.loop = asyncio.get_event_loop()
-
+        self.itchat = itchat.load_async_itchat()
```

Comment:
This method looks great, but I think it can be more concise:
```python
from itchat import load_async_itchat
...
self.itchat = load_async_itchat()
```
How do you think about it?

Reply:
It's more elegant than before, code has been pushed as required~